### PR TITLE
Add Haroun and the Sea of Stories by Salman Rushdie

### PR DESCRIPTION
**Pull request by elssar**
Most might not consider this, science fiction, but all of the "magic" in the book is controlled by science.

P2C2Es(Processes too Complicated to Explain) and M2C2D(Machines too Complicated to Describe) are behind all the "magic" and are under the supervision of the Eggheads of Earth's second moon, Kahani.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -148,6 +148,12 @@ A cast of strange and wonderful characters. Overarching themes on consciousness,
 
 > Set in the far future amidst a sprawling feudal interstellar empire where planetary dynasties are controlled by noble houses that owe an allegiance to the imperial House Corrino, Dune tells the story of young Paul Atreides (the heir apparent to Duke Leto Atreides & heir of House Atreides) as he & his family accept control of the desert planet Arrakis, the only source of the "spice" melange, the most important & valuable substance in the cosmos. The story explores the complex, multilayered interactions of politics, religion, ecology, technology & human emotion, as the forces of the empire confront each other for control of Arrakis. Published in 1965, it won the Hugo Award in 1966 & the inaugural Nebula Award for Best Novel. Dune is frequently cited as the world's best-selling sf novel.
 
+#### [Haroun and the Sea of Stories](https://www.goodreads.com/book/show/4835.Haroun_and_the_Sea_of_Stories) *by Salman Rushdie* [4.01]
```
Comment (sindresorhus) [inline readme.md]: We're rounding the rating: `[4.01]` => `[4.0]`

**Comment by sindresorhus** [comment]
@cassidoo @ocombe @davidmerrique @ruandre Thoughts?

**Comment by davidmerrique** [comment]
It's a bit of a stretch for me. I'd say no, but let's see what others have to say.

**Comment by cassidoo** [comment]
Hmm yeah this sounds a lot more like fantasy to me than sci-fi.

**Comment by ocombe** [comment]
Yes I agree, it's fantasy but not sci-fi

**Comment by hiddentao** [comment]
Agreed.

**Comment by elssar** [comment]
Alright, lets close it then.

**Pull request closed without merging**